### ISSUE TITLE
KeyError: 'releasedate'

### ISSUE THREAD

**Issue by josopu** [airship-david/Trailer-Downloader#51]
Some searches have no release date, it may be better to check if they exist or just handle an exception rather than quitting the script.

```
The Bourne Identity (2002)
Traceback (most recent call last):
  File "./download_all.py", line 115, in <module>
    main()
  File "./download_all.py", line 108, in main
    downloadItem()
  File "/mnt/user/Config/Trailer-Downloader-1.11/download.py", line 300, in main
    if arguments['year'].lower() in result['releasedate'].lower() and matchTitle(arguments['title']) == matchTitle(unescape(result['title'])):
KeyError: 'releasedate'
```
Edit:
Did happen again with other movie:
```
Police Academy 3 Back in Training (1986)
Traceback (most recent call last):
  File "./download_all.py", line 115, in <module>
    main()
  File "./download_all.py", line 108, in main
    downloadItem()
  File "/mnt/user/Config/Trailer-Downloader-1.11/download.py", line 303, in main
    if arguments['year'].lower() in result['releasedate'].lower() and matchTitle(arguments['title']) == matchTitle(unescape(result['title'])):
KeyError: 'releasedate'
```
But then I restarted the script and it didn't find the trailer but no errors where given. This may be related to Apple search.

**Comment by themoot** [comment]
Have the same error as well....

Babe (1995)
Traceback (most recent call last):
  File "/home/scripts/Trailer-Downloader-master/download_all.py", line 115, in <module>
    main()
  File "/home/scripts/Trailer-Downloader-master/download_all.py", line 108, in main
    downloadItem()
  File "/home/scripts/Trailer-Downloader-master/download.py", line 300, in main
    if arguments['year'].lower() in result['releasedate'].lower() and matchTitle(arguments['title']) == matchTitle(unescape(result['title'])):
KeyError: 'releasedate'

**Comment by war3zlod3r** [comment]
same error here Ip Man

Ip Man (2008)
Traceback (most recent call last):
  File "/home5/war3zlod3r/scripts/Trailer-Downloader/download_all.py", line 115, in <module>
    main()
  File "/home5/war3zlod3r/scripts/Trailer-Downloader/download_all.py", line 108, in main
    downloadItem()
  File "/home5/war3zlod3r/scripts/Trailer-Downloader/download.py", line 300, in main
    if arguments['year'].lower() in result['releasedate'].lower() and matchTitle(arguments['title']) == matchTitle(unescape(result['title'])):
KeyError: 'releasedate'

**Comment by airship-david** [comment]
This issue should be resolved in 1.12. Closing this for now but we can reopen it if the problem persists.
